src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,82 @@
+.nav {

```

Comment:
check paddings and margins here, maybe you should move flex content to the center
![image](https://user-images.githubusercontent.com/87998640/223975276-485f5112-f2ad-4270-80c7-51ea6fd55206.png)